src/app/views/home-view/home-view.component.ts

```diff
@@ -0,0 +1,164 @@
+})
+export class HomeViewComponent implements OnInit {
+
+  topAnime !: Anime[];
+  topManga !: Manga[];
+  thisSeason !: Anime[];
+  isLoading : boolean = true;
+
+  constructor(private apiService:ApiService, private route:ActivatedRoute) { }
+
+  ngOnInit(): void {
+    this.getTopAnime();
+    this.getTopManga();
+    this.getThisSeason();
+    this.topAnimeScroll();
+    this.topMangaScroll();
+    this.thisSeasonScroll();
+  }
+
+  getTopAnime(){
+    this.isLoading = true;
+    this.apiService.getTopAnime().subscribe(
+      {
+        next: (response:any) => {
+          this.topAnime = response.data;
+          console.log(this.topAnime);
+        },
+        error: (e) => {
+          this.isLoading = false;
+          //TODO: Redirect to not found
+          console.error(e);
+        },
+        complete: () => {
+          this.isLoading = false;
+        },
+      }
+    );
+  }
+
+  getTopManga(){
+    this.isLoading = true;
+    this.apiService.getTopManga ().subscribe(
+      {
+        next: (response:any) => {
+          this.topManga = response.data;
+          console.log(this.topManga);
+        },
+        error: (e) => {
+          this.isLoading = false;
+          //TODO: Redirect to not found
+          console.error(e);
+        },
+        complete: () => {
+          this.isLoading = false;
+        },
+      }
+    );
+  }
+
+  getThisSeason(){
+    this.isLoading = true;
+    this.apiService.getThisSeason().subscribe(
+      {
+        next: (response:any) => {
+          this.thisSeason = response.data;
+          console.log(this.thisSeason);
+        },
+        error: (e) => {
+          this.isLoading = false;
+          //TODO: Redirect to not found
+          console.error(e);
+        },
+        complete: () => {
+          this.isLoading = false;
+        },
+      }
+    );
+  }
+
+  topAnimeScroll(){
+    var carouselWidth = 8000;
+    var cardWidth = 600;
+    var scrollPosition = 0;
+
+    $("#nextAnimeButton").on("click", function () {
+      if (scrollPosition < (carouselWidth - cardWidth * 4)) { //check if you can go any further
+        scrollPosition += cardWidth;  //update scroll position
+        $("#carouselInnerTopAnime").animate({ scrollLeft: scrollPosition },600); //scroll left
+      }
+    });
+
+    $("#prevAnimeButton").on("click", function () {
+      if (scrollPosition > 0) {
+        scrollPosition -= cardWidth;
+        $("#carouselInnerTopAnime").animate(
+          { scrollLeft: scrollPosition },
+          600
+        );
+      }
+    });
+  }
+
+  topMangaScroll(){
+    var carouselWidth = 8000;
+    var cardWidth = 600;
+    var scrollPosition = 0;
+
+    $("#nextMangaButton").on("click", function () {
+      if (scrollPosition < (carouselWidth - cardWidth * 4)) { //check if you can go any further
+        scrollPosition += cardWidth;  //update scroll position
+        $("#carouselInnerTopManga").animate({ scrollLeft: scrollPosition },600); //scroll left
+      }
+    });
```

Comment:
De preferencia no usar jQuery, puesto que este binding de eventos ya lo provee Angular:

https://angular.io/guide/event-binding#binding-to-events

Por estar corriendo con la entrega, no es necesario cambiar esto, pero si tienes el tiempo estaría bien

Reply:
Se tendrá que mover a la siguiente entrega por temas de tiempo